sound/soc/sof/intel/hda.c

```diff
@@ -1776,6 +1776,12 @@ void hda_unregister_clients(struct snd_sof_dev *sdev)
 	hda_probes_unregister(sdev);
 }
 
+unsigned long hda_get_max_frequency(struct snd_sof_dev *sdev)
```

Comment:
I think you meant long long or u64, otherwise you have the ambiguity of unsigned long==u32 in some cases

Reply:
No, I meant `unsigned long` to be aligned with the clock API in case  one wants to re-use this value in any way outside.
```c
unsigned long clk_get_rate(struct clk *clk);
int clk_set_rate(struct clk *clk, unsigned long rate);
```
and friends.